src/contrib/cluster/Akka.Cluster.Sharding.Tests.MultiNode/ClusterShardingFailureSpec.cs

```diff
@@ -180,35 +181,51 @@ private void StartSharding()
                 shardResolver: extractShardId);
         }
 
-        [MultiNodeFact(Skip = "TODO")]
-        public void ClusterSharding_with_flaky_journal_should_setup_shared_journal()
+        [MultiNodeFact]
+        public void ClusterSharding_with_flaky_journal_network_specs()
+        {
+            ClusterSharding_with_flaky_journal_network_should_setup_shared_journal();
+            ClusterSharding_with_flaky_journal_network_should_join_cluster();
+            ClusterSharding_with_flaky_journal_network_should_recover_after_journal_network_failure();
+        }
+
+        public void ClusterSharding_with_flaky_journal_network_should_setup_shared_journal()
         {
             // start the Persistence extension
             Persistence.Persistence.Instance.Apply(Sys);
             RunOn(() =>
             {
-                Sys.ActorOf(Props.Create<MemoryJournal>(), "store");
-            }, _controller);
+                Persistence.Persistence.Instance.Apply(Sys).JournalFor("akka.persistence.journal.MemoryJournal");
```

Comment:
You may want to extract journal path into single constant field - it will make eventual changes easier.